src/stores/config.js

```diff
@@ -12,11 +12,13 @@ exports.setUp = function (basePath, blobStore, locks) {
           if (err) {
             return callback(err)
           }
+          var result
           try {
-            callback(null, JSON.parse(config))
```

Comment:
If JSON.parse throws, the error is automatically catch by the try/catch clause, here is a simplier example:

```JavaScript
try {
  checkIfRuns(JSON.parse('sdadadas:asda'))
} catch (e) {
  console.log('=> error')
}

function checkIfRuns (arg1) {
  console.log('->', arg1)
}
```

```bash
$ node try-catch.js
=> error
```